assets/src/apps/authoring/store/groups/layouts/deck/actions/validate.ts

```diff
@@ -79,8 +82,56 @@ const validateTarget = (target: string, activity: any, parts: any[]) => {
       return false;
   }
 };
+const checkExpressionsWithWrongBrackets = (value: string) => {
```

Comment:
this looks like a duplicate method of the one in scripting

Reply:
Correct. Fixed